### PR TITLE
Remove unwanted comma

### PR DESCRIPTION
**Pull request by NaveenDA**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
Closed and Reopened this PR in an attempt to attempt to break out of the Travis build pending status.

**Review by amandeepmittal — APPROVED** [review]
LGTM!

**Comment by amandeepmittal** [comment]
@RandellDawson is it okay to merge this PR?

File: curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md [hunk 1]
```diff
@@ -11,21 +11,21 @@ guideUrl: 'https://www.freecodecamp.org/guide/certificates/add-alt-text-to-an-im
...
 <code>&lt;img src=&quot;importantLogo.jpeg&quot; alt=&quot;Company logo&quot;&gt;</code>
 People with visual impairments rely on screen readers to convert web content to an audio interface. They won't get information if it's only presented visually. For images, screen readers can access the <code>alt</code> attribute and read its contents to deliver key information.
-Good <code>alt</code> text is short but descriptive, and meant to briefly convey the meaning of the image. You should always include an <code>alt</code> attribute on your image. Per HTML5 specification, this is now considered mandatory.
+Good <code>alt</code> text is short but descriptive and meant to briefly convey the meaning of the image. You should always include an <code>alt</code> attribute on your image. Per HTML5 specification, this is now considered mandatory.
```
Comment (RandellDawson) [inline curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md]: This should actually be written as:
```
Good <code>alt</code> text is short but descriptive and meant to briefly convey the meaning of the image. You should always include an <code>alt</code> attribute on your image. Per HTML5 specification, this is now considered mandatory.
```

File: curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md [hunk 2]
```diff
@@ -11,21 +11,21 @@ guideUrl: 'https://www.freecodecamp.org/guide/certificates/add-alt-text-to-an-im
...
 </section>
 
 ## Instructions
 <section id='instructions'>
-Camper Cat happens to be both a coding ninja and an actual ninja, and is building a website to share his knowledge. The profile picture he wants to use shows his skills, and should be appreciated by all site visitors. Add an <code>alt</code> attribute in the <code>img</code> tag, that explains Camper Cat is doing karate. (The image <code>src</code> doesn't link to an actual file, so you should see the <code>alt</code> text in the display.)
+Camper Cat happens to be both a coding ninja and an actual ninja and is building a website to share his knowledge. The profile picture he wants to use shows his skills and should be appreciated by all site visitors. Add an <code>alt</code> attribute in the <code>img</code> tag, that explains Camper Cat is doing karate. (The image <code>src</code> doesn't link to an actual file, so you should see the <code>alt</code> text in the display.)
```
Comment (RandellDawson) [inline curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md]: Actually, to really make this sentence more grammatically correct, it should read something like:

```
Camper Cat happens to be both a coding ninja and an actual ninja, who is building a website to share his knowledge. The profile picture he wants to use shows his skills and should be appreciated by all site visitors. 
```

File: curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md [hunk 3]
```diff
@@ -11,21 +11,21 @@ guideUrl: 'https://www.freecodecamp.org/guide/certificates/add-alt-text-to-an-im
...
 
 ## Tests
 <section id='tests'>
 
 ```yml
 tests:
-  - text: Your <code>img</code> tag should have an <code>alt</code> attribute, and it should not be empty.
-    testString: assert($('img').attr('alt'), 'Your <code>img</code> tag should have an <code>alt</code> attribute, and it should not be empty.');
+  - text: Your <code>img</code> tag should have an <code>alt</code> attribute and it should not be empty.
+    testString: assert($('img').attr('alt'), 'Your <code>img</code> tag should have an <code>alt</code> attribute and it should not be empty.');
```
Comment (RandellDawson) [inline curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md]: These changes look good.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@amandeepmittal, @NaveenDA  While a few of the commas needed to be removed, in some cases, an extra comma is needed.  See my requested changes.

**Review by RandellDawson — APPROVED** [review]

File: curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md (hunk 1 above)
Comment (RandellDawson) [reply curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md]: On second thought, I think two conjunctions in the same sentence makes it more complicated than needed.  Why not rephrase to:
`Good <code>alt</code> text provides the reader a brief description of the image.  You should always include an <code>alt</code> attribute on your image. Per HTML5 specification, this is now considered mandatory.`

**Review by amandeepmittal — APPROVED** [review]

File: curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md (hunk 3 above)
Comment (amandeepmittal) [reply curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md]: Yes, I agree.

File: curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md (hunk 1 above)
Comment (RandellDawson) [reply curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md]: @amandeepmittal What do you think about what I posted above?

File: curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md (hunk 2 above)
Comment (RandellDawson) [reply curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md]: @amandeepmittal What about my suggestion above?

**Comment by RandellDawson** [comment]
@amandeepmittal Can you take a look at the one last suggestion I had above?

File: curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md (hunk 2 above)
Comment (RandellDawson) [reply curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md]: On second thought, let's go ahead an approve the PR as is and I will create a new PR for the changes I have suggested, but the user has yet to add.

**Comment by RandellDawson** [comment]
Congrats @NaveenDA  on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**